CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
+# dbt_github v0.8.1  
+[PR #58](https://github.com/fivetran/dbt_github/pull/58) contains the following updates:
+
+## Bug Fixes 
+- Replaced the existing `dbt_current_timestamp.backcompat()` with the up-to-date `dbt_current_timestamp` macro to generate the current timestamp for customers. [PR #58](https://github.com/fivetran/dbt_github/pull/58)
```

Comment:
Can we add a bit more to this entry explaining why this was an issue in the past and how this updated addresses that bug.